Project_3/style.css

```
@@ -0,0 +1,240 @@
+/* ==============
+OVERALL DESIGN
+================ */
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #e5eeea;
+}
+
+/* ===========
+HEADER
+============== */
+
+header {
+    background-color: #ffffff;
+    height: 100vh;
+    margin-bottom: 100px;
+    padding: 15%;
+    width: 100%;
+}
+
+h1,
+h2 {
+    font-family: 'Abril Fatface', serif;
+}
+
+h1 {
+    color: #6bcad4;
+    font-size: 100px;
+    height: 400px;
+    width: 300px;
+}
+
+h2 {
+    color: #61935f;
+    font-size: 70px;
+}
+
+
+/* ==============
+VISUAL/PSUEDO ELEMENTS 
+=================*/
+
+body::before,
+body::after {
+    background-color: #24321b;
+    content: "";
+    position: fixed;
+}
+
+/* horizontal time line */
+body::before {
+    height: 25%;
+    margin-left: -10px;
+    margin-top: 40%;
+    width: 100%;
+    z-index: -3;
+}
+
+/* vertical time line */
+body::after {
+    bottom: 0;
+    height: 100vh;
+    left: 50%;
+    top: 0;
+    width: 10px;
+    z-index: -1;
+}
+
+
+/* =============
+TIMELINE CONTENT
+================ */
+
+img {
+    max-width: 100%;
+}
+
+figcaption,
+figure {
+    background-color: #ffffff;
+}
+
+figcaption {
+    padding: 0 5px 10px 5px;
```

The CSS will also work like this:
```suggestion
    padding: 0 5px 10px;
```
The `padding` shorthand works in a variety of ways; see [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) for some examples.